gradle.properties

```diff
@@ -4,7 +4,7 @@ pluginGroup=com.github.intheclouddan.intellijpluginld
 pluginName=intellij-plugin-ld
 pluginVersion=0.5.1-idea22
 pluginSinceBuild=213.6461.79
-pluginUntilBuild=221.*
+pluginUntilBuild=213.*
```

Comment:
our compatibility issues stem from 221 (unreleased) and i'm _fairly sure_ that it's related to the DSL pkg we're using being the 213 version not 221.

I think we should downgrade this for the time being

After talking with Dan, this might be a compelling reason to switch to the per year branch strategy